index.html

```diff
@@ -20,17 +20,92 @@ <h1>ToDoリスト</h1>
             <th>状態</th>
           </tr>
         </thead>
-        <tbody id="todolistarea"></tbody>
+        <tbody id="todolist_area"></tbody>
       </table>
-      <div id="textboxarea">
+      <div id="textbox_area">
         <p style="font-weight:bold;">新規タスクの追加</p>
-        <input id="task" type="text" name="task">
-        <input type="button" value="追加" id="btn">
+        <input id="newtask" type="text" name="task">
+        <input type="button" value="追加" id="newtask_btn">
       </div>
     </form>
 
     <script type="text/javascript">
 
+      const toDoListArea = document.getElementById('todolist_area');
+      const newTask = document.getElementById('newtask');
+      const newTaskBtn = document.getElementById('newtask_btn');
+      const taskList = [];
+
+      //todoを追加
+      newTaskBtn.addEventListener('click', () => {
+        const task = { taskName:newTask.value, taskStatus:'作業中' };
```

Comment:
修正漏れかと思いますが、`:`の後にもスペースを空けましょう。
```suggestion
        const task = { taskName: newTask.value, taskStatus: '作業中' };
```

エディタでファイル保存時に自動整形すると楽です。
https://qiita.com/mitashun/items/e2f118a9ca7b96b97840